src/ENS/ens.service.ts

```diff
@@ -128,21 +123,8 @@ export class EnsService {
         return this.publicResolver.name(node);
       }),
     );
-    const uniqDomains: Record<string, unknown> = {};
-    for (const item of domains) {
-      if (item && item.endsWith(domain) && !uniqDomains[item]) {
```

Comment:
@dwojno Because we're not filtering for subdomains anymore, will this change cache all of ENS?

Reply:
@jrhender nope, we're only getting domains that triggered event TextChanged and the text fields is 'metadata', so as far as I know, only iam-client-lib is adding this kind of text property to the ENS domain